site/pages/api/publish.test.ts

```diff
@@ -90,7 +90,7 @@ describe("publish", () => {
         expect(writeHeadMock).toBeCalledWith(302, { Location: ERROR_PATH });
     });
 
-    it.each([[disruptionWithConsequences], [disruptionWithNoConsequences]])(
+    it.each([[disruptionWithConsequences]])(
```

Comment:
Is "each" required as there is only one input

Reply:
done